cluster/deposit.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cluster
+
+// DepositData defines the deposit data to activate a validator.
+// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#depositdata
+type DepositData struct {
```

Comment:
q: any reason behind not going with `eth2p0.DepositData` type?

Reply:
I prefer not to couple our cluster definition with 3rd-party types. It should be exactly the same as `eth2p0.DepositData`.